lib/vagrant-service-manager/installer.rb

```diff
@@ -5,15 +5,15 @@ def initialize(type, machine, env, options)
         @type = type
         @machine = machine
         @env = env
-        @box_version = options[:box_version]
+        @options = { type: @type }.merge(options)
```

Comment:
Alternatively, if _options_ is really just the options passed to the CLI and _type_ something you determine, keep passing both parameters along.